### PR TITLE
Protectli PTx01 test & hardware matrix

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemyslaw Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/variants/protectli_ptx01/hardware-matrix.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| Component                      | Description                                 |
+|--------------------------------|---------------------------------------------|
+| **CPU**                        | Intel(R) Celeron(R) N5105 @ 2.00GHz         |
+| **RAM**                        | Micron MT53D512M64D4RQ-046                  |
+| **Flash memory**               | Winbond 25Q16JVSSIQ                         |
+| **USB Keyboard**               | DELL KB216                                  |
+| **USB Mouse**                  | DELL MS116p                                 |
+| **USB pendrives**              | 1. SanDisk Ultra  Flair USB 3.0 16 GB       |
+| **Display**                    | HDMI 1920x1080p                             |
+| **Ethernet controller**        | 2x intel i225 (on-board)                    |
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/variants/protectli_ptx01/hardware-matrix.md]: improved in 26490ebf4193cd182621ffdb07ea17a9578f17d0

File: docs/variants/protectli_ptx01/test-matrix.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+The test matrix is used to determine the scope of tests which the DUT is
+subjected from before the release of the new binary. The test scope is the same
+for all platforms of the PT family.
+
+## Module: Dasharo compatibility
+
+| No.  | Supported test suite                  | Test suite ID | Supported test cases                 |
+|:----:|:--------------------------------------|:-------------:|:-------------------------------------|
+| 1.   | [Memory HCL][HCL]                     | HCL           | All                                  |
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/variants/protectli_ptx01/test-matrix.md]: improved in e1518d70e96e011449816ce4df3c81a35c705a36

File: docs/variants/protectli_ptx01/test-matrix.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+| 2.   | [UEFI compatible interface][EFI]      | EFI           | All                                  |
+| 3.   | [Display ports][DSP]                  | DSP           | DSP002.001, DSP002.002, DSP002.003   |
+| 4.   | [Network boot utilities][NBT]         | NBT           | All                                  |
+| 5.   | [NVMe support][NVM]                   | NVM           | All                                  |
+| 6.   | [Custom logo][CLG]                    | CLG           | All                                  |
+| 7.   | [Custom Boot Keys][CBK]               | CBK           | All                                  |
+| 8.   | [USB HID and MSC Support][USB]        | USB           | USB001.xxx and USB002.xxx            |
+| 9.   | [Debian Stable and Ubuntu LTS support][LBT]  | LBT    | LBT003.001, LBT003.002, LBT004.001, LBT004.002|
+| 10.  | [USB-C/Thunderbolt][UTC]              | UTC           | UTC004.001, UTC004.002               |
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/variants/protectli_ptx01/test-matrix.md]: improved in e1518d70e96e011449816ce4df3c81a35c705a36

File: docs/variants/protectli_ptx01/test-matrix.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+| 11.  | [M.2 WiFi/Bluetooth][WLE]             | WLE           | All                                  |
+| 12.  | [eMMC support][MMC]                   | MMC           | All                                  |
+| 13.  | [SMBIOS][DMI]                         | DMI           | DMI002.001, DMI003.001, DMI004.001, DMI005.001, DMI006.001 |
+| 14.  | [Custom network boot entries][CNB]    | CNB           | CNB001.002                           |
+| 15.  | [Audio subsystem][AUD]                | AUD           | AUD007.xxx, AUD008.xxx               |
+| 16.  | [UEFI Shell][USH]                     | USH           | All                                  |
+| 17.  | [USB detection][UDT]                  | UDT           | All                                  |
+| 18.  | [USB booting][UBT]                    | UBT           | All                                  |
+| 19.  | [Windows booting][WBT]                | WBT           | WBT001.001                           |
+
```
[the remark this answers is not in the document]
Comment (sulewskiprzemyslaw) [reply docs/variants/protectli_ptx01/test-matrix.md:31]: improved in e1518d70e96e011449816ce4df3c81a35c705a36

**Comment by miczyg1** [comment]
LGTM for starters

**Pull request merged**